business-central-parent/business-central-distribution-wars/business-monitoring/pom.xml

```
@@ -31,7 +31,7 @@
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>business-monitoring-webapp</artifactId>
-      <classifier>thorntail</classifier>
+      <classifier>bootable</classifier>
```

could we take standalone classifier ?
```suggestion
      <classifier>standalone</classifier>
```

---

was this forgotten to apply? as I can see below it already uses standalone classifier?